### ISSUE TITLE
Installed new extensions do not appear after installation, but appear after restarting the application.

### ISSUE THREAD

**Issue by rodriguezrobinj** [mihonapp/mihon#246]
### Steps to reproduce

1. Download new extension 
3. Install new extension

### Expected behavior

When installing the extension it does not appear installed, but only after restarting the application.

### Actual behavior

When installing the extension it does not appear installed, but only after restarting the application.

### Mihon version

0.16.2

### Android version

Android 12

### Device

Tecno Spark 10c

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.1](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.
